testsuite/MDAnalysisTests/test_auxiliary.py

```diff
@@ -35,7 +35,12 @@ def __init__(self):
 class BaseAuxReaderTest(object):
     def __init__(self, reference):
         self.ref = reference
-        self.reader = self.ref.reader('test', self.ref.testdata)
+
+    def setUp(self):
```

Comment:
http://nose.readthedocs.io/en/latest/writing_tests.html#test-classes

This actually says it calls both for each test. Since a new instance is created and setUp is called on that instance. So mixing is not a problem for running the tests just a style issue.